src/components/gridList/useGridLayout.ts

```diff
@@ -16,7 +16,7 @@ const useGridLayout = (props: GridListBaseProps) => {
     keepItemSize,
     containerWidth,
     contentContainerStyle
-  } = props;
+  } = useThemeProps(props, 'useGridLayout');
```

Comment:
I commented about it in the private PR. If we don't have any alternatives let's go for it..